org.feichtmeier.Musicpod.json

```diff
@@ -0,0 +1,228 @@
+                                   "x-checker-data": {
+                                        "type": "anitya",
+                                        "project-id": 9265,
+                                        "stable-only": true,
+                                        "url-template": "https://www.freedesktop.org/software/uchardet/releases/uchardet-$version.tar.xz"
+                                   }
+                              }
+                         ]
+                    },
+                    {
+                         "name": "squashfs-tools",
+                         "cleanup": ["*"],
+                         "buildsystem": "simple",
+                         "build-commands": [
+                              "XZ_SUPPORT=1 LZO_SUPPORT=0 make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=${FLATPAK_DEST}/bin INSTALL_MANPAGES_DIR=${FLATPAK_DEST}/man/man1"
+                         ],
+                         "sources": [
+                              {
+                                   "type": "git",
+                                   "url": "https://github.com/plougher/squashfs-tools.git",
+                                   "tag": "4.6.1",
+                                   "commit": "d8cb82d9840330f9344ec37b992595b5d7b44184",
+                                   "x-checker-data": {
+                                        "type": "git",
+                                        "tag-pattern": "^([\\d.]+)$"
+                                   }
+                              }
+                         ]
+                    },
+                    {
+                         "name": "musicpod",
+                         "buildsystem": "simple",
+                         "build-commands": [
+                              "unsquashfs -quiet -no-progress musicpod.snap",
+                              "cp -r squashfs-root/bin /app",
+                              "install -Dm644 squashfs-root/snap/gui/musicpod.desktop /app/share/applications/org.feichtmeier.Musicpod.desktop",
+                              "desktop-file-edit --set-icon='org.feichtmeier.Musicpod' /app/share/applications/org.feichtmeier.Musicpod.desktop",
+                              "install -Dm644 squashfs-root/snap/gui/musicpod.png /app/share/icons/hicolor/256x256/apps/org.feichtmeier.Musicpod.png",
+                              "install -Dm644 org.feichtmeier.Musicpod.appdata.xml /app/share/metainfo/org.feichtmeier.Musicpod.appdata.xml"
+                         ],
+                         "cleanup": [
+                              "/include",
+                              "/lib/pkgconfig",
+                              "/share/man"
+                         ],
+                         "sources": [
+                              {
+                                   "type": "file",
+                                   "sha256": "aec5891c54a81ab64799b1e9efa05899d2ae16503313168152dae80f25523ce2",
```

Comment:
```suggestion
                                   "sha256": "627b91c84817e75f34bcc33b5c5aeb3c29ae7f2d35d64f7b08c8022800d32ec4",
```

Reply:
doesnt this need to match the sha from above? or is this a different file?

Reply:
I don't think show. Each sha256 is for a different file. This one is for the XML file, the other two are for the x86 and arm snaps.

Reply:
I just did what you did and got this hash: aec5891c54a81ab64799b1e9efa05899d2ae16503313168152dae80f25523ce2

I'm new to this hash stuff, so if you're confident that you know how to get a hash, tell me and I can use that.

Reply:
That looks like the old hash, if you changed the file, the hahs must change too.

Are you using the new file with the commit from here?

Reply:
Yes I'm using the link in the line below, and yes it has the changes we made.  I entered
`wget https://raw.githubusercontent.com/ubuntu-flutter-community/musicpod/125f383f759e5d6e35f77ced2f8f65c853072305/flatpak/org.feichtmeier.Musicpod.appdata.xml
`

then
`sha256sum org.feichtmeier.Musicpod.appdata.xml`

And get

aec5891c54a81ab64799b1e9efa05899d2ae16503313168152dae80f25523ce2

Reply:
Then use it, if it's wrong the CI will tell us.

Reply:
<img width="833" alt="Bildschirmfoto 2024-04-25 um 08 24 40" src="https://github.com/flathub/flathub/assets/15329494/5c06d196-2ecc-4aac-ba39-739664263fa3">
looks like its outdated, maybe to avoid additional noise onto the maintainers here lets talk 

available after 17 cet, discord server: https://discord.gg/vJgPV9Yn3n

Reply:
I'm not sure if that's necessary. If the Sha256 hasn't changed, and I've tried to double-check the sha256, there's nothing to be done except see if the CI tells us its wrong. @bbhtt, I think we're ready to check.